examples/color/colormap_reference.py

```diff
@@ -67,7 +62,17 @@ def plot_color_gradients(cmap_category, cmap_list):
 for cmap_category, cmap_list in cmaps:
     plot_color_gradients(cmap_category, cmap_list)
 
-plt.show()
+
+###############################################################################
+# Reversing a colormap
+# --------------------
+#
+# A reversed version of any colormap is available by appending ``_r`` to the
+# name, for example:
+
+plot_color_gradients("Reversed ",
```

Comment:
How about showing viridis and viridis_r in direct comparison instead of some random maps where I have to look up the unreversed version above to see the effect?